docs/source/en/main_classes/output.mdx

```diff
@@ -136,6 +136,30 @@ documented on their corresponding model page.
 
 [[autodoc]] modeling_outputs.Seq2SeqQuestionAnsweringModelOutput
 
+## SemanticSegmenterOutput
```

Comment:
Think they weren't added here before cc @NielsRogge @sgugger

Reply:
Thanks for adding them!